crates/viona-api/sys/src/lib.rs

```diff
@@ -67,6 +68,14 @@ mod structs {
         pub vrs_qsize: u16,
         pub vrs_qaddr: u64,
     }
+
+    #[repr(C)]
+    pub enum vioc_promisc {
+        VIONA_PROMISC_NONE = 0,
+        VIONA_PROMISC_MULTI,
+        VIONA_PROMISC_ALL,
+        VIONA_PROMISC_ALL_VLAN,
```

Comment:
The `falcon` feature should be painted through into `viona_api_sys`, and the `ALL_VLAN` variant hidden with `cfg()` until such time as it's available in at least stlouis.

Reply:
Done.  I also rebased.  I don't usually do that mid-review, since it makes it harder to see incremental changes, but the total diff is small enough that it's not hard to keep track of.